lib/ruby_lsp/handler.rb

```diff
@@ -62,7 +62,20 @@ def handle(request)
             error = e
           end
 
-          @writer.write(id: request[:id], result: result) unless result == VOID
+          if error
+            @writer.write(
+              {
+                id: request[:id],
+                error: {
+                  code: LanguageServer::Protocol::Constant::ErrorCodes::INTERNAL_ERROR,
+                  message: error.inspect,
+                  data: request.to_s,
```

Comment:
If we don't turn `request` into String here, it'll be shown as `[Object object]` in the output.